react/features/base/sounds/reducer.js

```diff
@@ -0,0 +1,106 @@
+// @flow
+import {assign, ReducerRegistry} from '../redux';
+
+import {
+    _ADD_AUDIO_ELEMENT,
+    _REMOVE_AUDIO_ELEMENT, REGISTER_SOUND, UNREGISTER_SOUND
+} from './actionTypes';
+import type { AudioElement } from '../media';
+
+const logger = require('jitsi-meet-logger').getLogger(__filename);
+
+type Sound = {
+    audioElement?: AudioElement,
+    soundId: String,
+    src: Any
+}
+
+/**
+ * Initial state of the sounds reducer which is a {@code Map} of globally stored
+ * sounds.
+ *
+ * @type {Map<string, Sound>}
+ */
+const SOUNDS_INITIAL_STATE = new Map();
+
+/**
+ *
+ */
+ReducerRegistry.register(
+'features/base/sounds', (state = SOUNDS_INITIAL_STATE, action) => {
+    switch (action.type) {
+    case _ADD_AUDIO_ELEMENT:
+    case _REMOVE_AUDIO_ELEMENT: {
+        return _addOrRemoveAudioElement(state, action);
+    }
+    case REGISTER_SOUND: {
+        const newSoundsMap = new Map(state);
```

Comment:
I did not do that only, because they are one liners (with debug message removed) and then I would have to write extra docs for each method :) And it was not a regular convention or maybe it came from old code... (see switch in Participants).

Reply:
But I will change that given there will be methods for add/remove audio element.